services/dynamic-sidecar/Dockerfile

```diff
@@ -52,6 +52,16 @@ RUN \
   --mount=type=bind,source=scripts/install_rclone.bash,target=install_rclone.bash \
   ./install_rclone.bash
 
+RUN AWS_CLI_VERSION="2.11.11" \
+    && curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${AWS_CLI_VERSION}.zip" -o "awscliv2.zip" \
+    && apt-get update && apt-get install -y unzip \
+    && unzip awscliv2.zip \
+    && ./aws/install \
+    && apt-get remove --purge -y unzip \
+    && rm awscliv2.zip \
+    && rm -rf awscliv2 \
+    && aws --version
+
```

Comment:
is --purge the same as ```apt-get clean -y```?

Reply:
- `apt-get clean` - his command is used to clear out the local repository of retrieved package files. It removes everything except the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.
- `-- purge` - The --purge option is used with the apt-get remove or apt-get autoremove commands. It removes packages along with their configuration files.